src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -115,9 +120,10 @@ def __get_user_activities() -> Optional[pd.DataFrame]:
     return __get_user_activities
 
 
-def _get_submissions() -> list:
-    assignments_df: pd.DataFrame = result_bucket["assignments"]
-    return service.get_submissions(assignments_df)
+def _get_submissions() -> pd.DataFrame:
+    assigment_id: int = shared_data["current_assignment_id"]
+    section_id: int = shared_data["current_section_id"]
```

Comment:
The pattern used elsewhere is to pass these values as arguments instead of relying on cached values.

Reply:
I just changed the logic to make it work like the rest of methods